mantle/platform/api/gcloud/compute.go

```diff
@@ -147,9 +147,9 @@ func (a *API) mkinstance(userdata, name string, keys []*agent.Key, opts platform
 		})
 	}
 	// create confidential instance
-	if a.options.Confidential {
+	if a.options.ConfidentialType == "SEV" || a.options.ConfidentialType == "SEV_SNP" {
```

Comment:
Maybe let's support users typing the more usual "SEV-SNP"... or the lowercase version of those too?

Reply:
SGTM, now the input like `sev, sev_snp, sev-snp`, then will convert lowercase characters to uppercase and also replace `-` to `_`, should be `SEV, SEV_SNP` finally.